src/vue/status-bar/port-utilization.js

```diff
@@ -0,0 +1,25 @@
+import Stat from "./stat.js";
+
+const PortUtilization = {
+    props: ["usageDown", "usageUp"],
+    components: {
+        stat: Stat,
+    },
+    // a bit of a hack here to get around the current translations.
+    // vue i18n provides slightly different api for this. But
+    // it's also possible to provide custom formatter
+    filters: {
+        stripEnd(value) {
+            return value.replace(/\$1%/, "");
+        },
```

Comment:
This hack needs to be fixed in a better way:
- One solution is to "migrate" the messages phrases to be compatible. The only problem with this is that all the translators will need to change the translations too, and this is a big work.
- The "hack" must be only at one place. Now you are adding this to any component with the problem, this is not a good way.
- I'm not too sure what this replacement does exactly.  Can you explain better what the problem is with one example?

Reply:
I think the "best" solution would be to just use appropriate formatter for i18nvue. Default one uses `{token}` syntax, but it's possible to just write custom one which would use the same syntax as `i18nnext`.

about multiple places. It's possible to register filter with single component or just the instance. Could easily pull this out if needed to just the status bar.

Messages had `CPU load: $1%` in them, this strips that end part so that you don't get `CPU load: $1%20%` etc. in status bar.

(more info about custom formatters)[https://kazupon.github.io/vue-i18n/guide/formatting.html#custom-formatting]